Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceRenderer.swift

```diff
@@ -314,7 +314,7 @@ internal class ExperienceRenderer: ExperienceRendering, StateMachineOwning {
 
     /// Reset only the owned `Self.stateMachine` instance in conformance to `StateMachineOwning`.
     func reset() {
-        stateMachine?.removeObserver(analyticsObserver)
+        stateMachine?.removeAnalyticsObserver()
```

Comment:
Changed this because the AppcuesFrameView doesn't have a reference to the instance to use the previous `removeObserver`.